gen/ivory.hoon

```diff
@@ -1,56 +1,61 @@
 ::
-::::  /hoon/metal/gen
+::::  /hoon/ivory/gen
   ::
 /?    310
 ::
 ::::
   !:
 :-  %say
-|=  $:  {now/@da * bec/beak}
-        *
-    ==
+|=  [[now=@da * bec=beak] *]
 :-  %noun
-::
 ::  sys: root path to boot system, `/~me/[desk]/now/sys`
 ::
-=+  sys=`path`/(scot %p p.bec)/[q.bec]/(scot %da now)/sys
-::
+=/  sys=path
+  /(scot %p p.bec)/[q.bec]/(scot %da now)/sys
 ::  compiler-source: hoon source file producing compiler, `sys/hoon`
 ::
-=+  compiler-source=.^(@t %cx (welp sys /hoon/hoon))
-::
+=/  compiler-source
+  .^(@t %cx (welp sys /hoon/hoon))
 ::  compiler-hoon: compiler as hoon expression
 ::
+::    We parse with +ream for reproducibility.
+::
 ~&  %ivory-parsing
-=+  compiler-hoon=(ream compiler-source)
+=/  compiler-hoon  (ream compiler-source)
```

Comment:
I think this should be `(rain /hoon/hoon compiler-source)`.

Reply:
Arvo too?